projects/chaos/inc/chaos_config.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "gpio.h"
+#include "power_path.h"
+
+typedef struct ChaosConfig {
+  PowerPathCfg power_path;
+  // Roof
+  const GPIOAddress telemetry_power;
+  const GPIOAddress array_sense_power;
+  const GPIOAddress rear_camera_power;
+  // Front
+  const GPIOAddress themis_power;
```

Comment:
We split driver display and lights? I was under the impression that the front just received protected/unprotected.